recipes/bzip2/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+if [ `uname` == Darwin ]; then
+    make install PREFIX=$PREFIX CFLAGS="$CFLAGS"
+else
+    make -f Makefile-libbz2_so
```

Comment:
That in the standard `Makefile`. Now the `build.sh` file does both `make` and `makefile -f Makefile-libbz2_so`.